fe/pages/dashboards.py

```diff
@@ -11,6 +14,121 @@
     icon="bi-bar-chart-line",
 )
 
-layout = _iframe.layout(
-    "https://lookerstudio.google.com/embed/reporting/86ab32f9-151b-4f91-87eb-060a22f2f890/page/QJFZE"
+DEPMAP_DATA = pd.read_parquet(
+    os.path.abspath("./") + "/parquet_data_sources" + "/depmap_metadata.parquet")
+ONCTOTREE_LINEAGE_VALUES = DEPMAP_DATA["OncotreeLineage"].unique()
+
+MAVEDB_DATA = pd.read_parquet(
+    os.path.abspath("./") + "/parquet_data_sources" + "/mavedb_metadata.parquet")
```

Comment:
It would be useful to have instructions as to how generate `depmap_metadata.parquet` and `mavedb_metadata.parquet`, so that we could update or replace them in the future

Reply:
Discussed this in Slack - will need to have this logic in Apache Airflow